src/shared.js

```diff
@@ -2748,6 +2749,20 @@ export const checkOutParticipant = async (data) => {
     }
 };
 
+export const participantCanCheckIn = (data) => {
+    let yesString = conceptIds.yes + '';
+    if(
+        (data[conceptIds.withdrewConsent] + '') === yesString ||
+        (data[conceptIds.revokedHIPAA] + '') === yesString ||
+        (data[conceptIds.destroyData] + '') === yesString ||
+        (data[conceptIds.dataDestroyed] + '') === yesString
```

Comment:
From lines 2753 to 2758, I think there's no need to convert concept ID of "yes" to strings.

Reply:
Done. Look good now?

Reply:
Good.
A shorter version can be:
```js
return data[conceptIds.withdrewConsent] !== conceptIds.yes &&
        data[conceptIds.revokedHIPAA] !== conceptIds.yes &&
        data[conceptIds.destroyData] !== conceptIds.yes &&
        data[conceptIds.dataDestroyed] !== conceptIds.yes;
```

Reply:
Done. All set?